samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -158,9 +159,26 @@ public async Task<string> ExtractChatHistoryAsync(SKContext context)
             var formattedMessage = chatMessage.ToFormattedString();
             var tokenCount = Utilities.TokenCount(formattedMessage);
 
-            // Plan object is not meaningful content in generating chat response, exclude it
-            if (remainingToken - tokenCount > 0 && !formattedMessage.Contains("proposedPlan\\\":", StringComparison.InvariantCultureIgnoreCase))
+            if (remainingToken - tokenCount > 0)
```

Comment:
I'm not gonna lose sleep over this, but shouldn't it be >= instead of simply >?

Reply:
Good catch!